apps/novel-builder/src/panels/index.tsx

```diff
@@ -36,6 +37,12 @@ function PanelExplorer() {
   const dispatch = useAppDispatch();
   const { openModal: openAreYouSure } = AreYouSure.useAreYouSure();
 
+  useEffect(() => {
+    if (!window.onbeforeunload) {
```

Comment:
osea solo tiene que importar si la novela esta definida, sino se agrega siempre al principio. lo testeaste de cerrarlo sin hacer ningun cambio?
```suggestion
    if (!window.onbeforeunload && novel) {
```

Reply:
quiza este componente no se renderiza hasta que le cargas o empezas una novela